functional-tests/CloudbaseInit.Tests.ps1

```diff
@@ -287,6 +301,29 @@ function after.cloudbaseinit.plugins.windows.bootconfig.BootStatusPolicyPlugin {
 
 function prepare.nocloud {
     pushd "$here/../$($env:CLOUD)"
+
+        $openVpnUrl = "https://build.openvpn.net/downloads/releases/OpenVPN-2.5.10-I601-amd64.msi"
+        $wc = New-Object System.Net.WebClient
+        $msiFilePath = Join-Path $(pwd) "openvpn.msi"
+        $wc.DownloadFile($openVpnUrl, $msiFilePath)
+        cmd /c "msiexec.exe -i ${msiFilePath} /qn /norestart /l*v test.log"
+        Start-Sleep 10

```

Comment:
Let's try to avoid the sleep and instead wait for the job to complete:

https://stackoverflow.com/questions/38481952/make-msiexec-wait-until-the-installation-completes